### PR TITLE
Tw 732 load more in add member in group need support mouse scroll

### PR DESCRIPTION
**Pull request by drminh2807**
# Issue
- #732 
- #679 

# Demo

https://github.com/linagora/twake-on-matrix/assets/12546908/0950efce-6e00-4826-b02e-e24dddfa597a

https://github.com/linagora/twake-on-matrix/assets/12546908/eff851e7-0f62-430c-b414-eb1fc598ce8b

### CODE REVIEW

File: lib/widgets/twake_components/twake_smart_refresher.dart [hunk 1]
```diff
@@ -1,59 +1,131 @@
...
+  void onRefresh() {
+    refreshNotifier.value = true;
+  }
+
+  void refreshCompleted() {
+    refreshNotifier.value = false;
+  }
+
+  void onLoading() {
+    loadNotifier.value = true;
```
Comment (sherlockvn) [inline lib/widgets/twake_components/twake_smart_refresher.dart:40]:
```suggestion
    loadingNotifier.value = true;
```

File: lib/widgets/twake_components/twake_smart_refresher.dart [hunk 2]
```diff
@@ -1,59 +1,131 @@
...
       ),
-      controller: controller,
-      onRefresh: onRefresh,
-      onLoading: onLoading,
-      child: child,
+    );
+  }
+}
+
+class _LoadingIndicator extends StatelessWidget {
```
Comment (sherlockvn) [inline lib/widgets/twake_components/twake_smart_refresher.dart:121]: this appear in alot of places, can you twake-ize it for all loading indicator ?

Comment (drminh2807) [reply lib/widgets/twake_components/twake_smart_refresher.dart]: done

File: lib/pages/contacts_tab/contacts_tab_body_view.dart [hunk 3]
```diff
@@ -16,78 +18,87 @@ class ContactsTabBodyView extends StatelessWidget {
...
+    if (controller.refreshController == null) return const SizedBox();
+    return TwakeSmartRefresher(
+      onLoading: controller.loadMoreContacts,
+      controller: controller.refreshController!,
+      slivers: [
+        SliverToBoxAdapter(
+          child: Divider(
+            height: 1,
+            thickness: 1,
+            color: Colors.black.withOpacity(0.15),
```
Comment (sherlockvn) [inline lib/pages/contacts_tab/contacts_tab_body_view.dart:30]: move to style ?

**Review by sherlockvn — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Review by imGok — APPROVED** [review]

**Pull request merged**